esmvaltool/references/massonet18natcc.bibtex

```diff
@@ -0,0 +1,17 @@
+@Article{Massonnet2018,
```

Comment:
```suggestion
@Article{massonet18natcc,
```
It has to be consistent with the reference tag.

Reply:
Sorry, I forgot to modify that.